browser/js/config/configureStore.js

```diff
@@ -8,12 +8,13 @@ import rootReducer from '../reducers';
 const loggerMiddleware = createLogger();
 
 export default function configureStore (initialState) {
+  const middlewares = [thunkMiddleware]
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
When Webpack builds our code for production it will address this correctly, I tested it by telling webpack to build for production and there were not log messages other than our missed console.logs(). It was required that we added an extra webpack plugin into our webpack config file though.